mapviz/src/map_canvas.cpp

```diff
@@ -28,9 +28,9 @@
 // *****************************************************************************
 
 
-#include <GL/glew.h>
-#include <GL/gl.h>
-#include <GL/glu.h>
+//#include <GL/glew.h>
```

Comment:
Probably  better to remove these (and other similar lines) instead of just commenting them out.

Reply:
Will do

Reply:
Need to fix some other things that were commented out for testing as well